packages/components/src/components/Text/Text.stories.tsx

```diff
@@ -52,6 +56,9 @@ export const Body: Story = {
 const children = 'Lorem ipsum dolor sit amet.'
 
 export const _Heading: Story = {
+  argTypes: {
+    tone: { control: 'radio', options: ['default', 'subtle', 'inverse'] },
```

Comment:
Be a slight improvement if the options could be pulled directly from HeaderProps.tone or BaseTones so if any are ever added/removed it doesn't break the story.

Reply:
TypeScript doesn't allow you to build an array from a type, but you can do the opposite. Declared and exported an array called `baseToneValues` in `Text.tsx` and updated the type to use that. The syntax is a little weird but solves the problem of hardcoding the options.